### PR TITLE
Make the instructions more consistent in the browser section

### PR DESCRIPTION
**Pull request by mfwmyfacewhen**
The configuration guides for the browser section were inconsistent so I fixed it.

### CODE REVIEW

File: docs/browsers.en.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ These options can be found in :material-menu: → **Settings** → **Privacy & S
 
 ##### Enhanced Tracking Protection
 
-- [x] Enable **Strict** Enhanced Tracking Protection
+- [x] Select **Strict** Enhanced Tracking Protection
```
Comment (subatomic-honda-civic) [inline docs/browsers.en.md:63]: Since you're using "Disable" in place of "Uncheck", maybe perhaps use "Enable"? (Select/Disable or Enable/Disable)

**Review by subatomic-honda-civic — CHANGES_REQUESTED** [review]

File: docs/browsers.en.md (hunk 1 above)
Comment (mfwmyfacewhen) [reply docs/browsers.en.md:63]: I was using enable/disable for toggles, Select for those bulletpoint menu things, and check/uncheck for checkboxes. I thought "Select" made more sense for that option sense you have to select one of the options, you can't enable/disable it.

Comment (subatomic-honda-civic) [reply docs/browsers.en.md:63]: > I was using enable/disable for toggles, Select for those bulletpoint menu things, and check/uncheck for checkboxes. I thought "Select" made more sense for that option sense you have to select one of the options, you can't enable/disable it.


Makes more sense, my last brain cells were probably failing already :smile:

**Review by subatomic-honda-civic — APPROVED** [review]

**Review by freddy-m — APPROVED** [review]
LGTM

**Pull request merged**